bench/tx-generator/src/Cardano/Benchmarking/Compiler.hs

```diff
@@ -250,3 +260,46 @@ newWallet n = do
   name <- WalletName <$> newIdentifier n
   emit $ InitWallet name
   return name
+
+parseKey :: BSL.ByteString -> TextEnvelope
+parseKey x = case decode x of
+  Nothing -> error $ "parsing of key failed : " ++ show x
+  Just k -> k
+
+keyNameGenesisInputFund :: KeyName
+keyNameGenesisInputFund = KeyName "GenesisInputFund"
+
+keyNameTxGenFunds :: KeyName
+keyNameTxGenFunds = KeyName "TxGenFunds"
+
+-- addr_test1vzd3muund27y5nw83vymqj3a83pcuzkkejej6s75e5lfjcc85nc3p
+keyTxGenFunds :: TextEnvelope
+keyTxGenFunds = parseKey "{\n    \"type\": \"PaymentSigningKeyShelley_ed25519\",\n    \"description\": \"Payment Signing Key\",\n    \"cborHex\": \"5820617f846fc8b0e753bd51790de5f5a916de500175c6f5a0e27dde9da7879e1d35\"\n}\n"
+
+keyNameSplitPhase :: KeyName
+keyNameSplitPhase = KeyName "SplitPhase"
+
+-- addr_test1vz45dtkyzk6s3245qw8hmaddaatcx8td3pvmntl8ty7q99c22eahm
+keySplitPhase :: TextEnvelope
+keySplitPhase = parseKey "{\n    \"type\": \"PaymentSigningKeyShelley_ed25519\",\n    \"description\": \"Payment Signing Key\",\n    \"cborHex\": \"5820cf0083c2a5d4c90ab255bc8e68f407d52eebd9408de60a0b9e4c468f9714f076\"\n}\n"
+
+-- addr_test1vzj7zv9msmdasvy5nc9jhnn2gqvrvu33v5rlg332zdfrkugklxkau
+keyNameBenchmarkInputs :: KeyName
+keyNameBenchmarkInputs = KeyName "BenchmarkInputs"
+
+keyBenchmarkInputs :: TextEnvelope
+keyBenchmarkInputs = parseKey "{\n    \"type\": \"PaymentSigningKeyShelley_ed25519\",\n    \"description\": \"Payment Signing Key\",\n    \"cborHex\": \"58205b7f272602661d4ad3d9a4081f25fdcdcdf64fdc4892107de50e50937b77ea42\"\n}\n"
+
+keyNameBenchmarkDone :: KeyName
+keyNameBenchmarkDone = KeyName "BenchmarkingDone"
+
+-- addr_test1vz4qz2ayucp7xvnthrx93uhha7e04gvxttpnuq4e6mx2n5gzfw23z
+keyBenchmarkDone :: TextEnvelope
+keyBenchmarkDone = parseKey "{\n    \"type\": \"PaymentSigningKeyShelley_ed25519\",\n    \"description\": \"Payment Signing Key\",\n    \"cborHex\": \"582016ca4f13fa17557e56a7d0dd3397d747db8e1e22fdb5b9df638abdb680650d50\"\n}\n"
+
+keyNameCollaterals :: KeyName
+keyNameCollaterals = KeyName "Collaterals"
+
+-- addr_test1vpckd9muw3l4f8ne4uzumy28p0k84rvx48q46kssjkta5ng4v6sfs
+keyCollaterals :: TextEnvelope
+keyCollaterals = parseKey "{\n    \"type\": \"PaymentSigningKeyShelley_ed25519\",\n    \"description\": \"Payment Signing Key\",\n    \"cborHex\": \"58204babdb63537ccdac393ea23d042af3b7c3587d7dc88ed3b66c959f198ad358fa\"\n}\n"
```

Comment:
Would it make sense to add a (very brief) human-readable [Note] on how the flow through these wallets is designed during a benchmarking run? I know the names speak for themselves, but it would help creating specific queries for `cardano-cli`.

Reply:
Done that.